src/main/webapp/i18n/de/quizExercise.json

```
@@ -261,6 +262,12 @@
                 "individual": "Einzeln",
                 "explanation": "Ausführungsmodus bestimmt wie Studierende an einem Quiz teilnehmen können.\n\nSynchronisiert: Alle Studierenden nehmen gleichzeitig an dem Quiz teil.\nGruppiert: Es gibt mehrere Gruppen an denen Studierende mit einem Passwort teilnehmen können.\nEinzeln: Studierende können jederzeit selbstständig an dem Quiz teilnehmen."
             },
+            "releaseDateExplanation": {
+                "synchronized": "Synchronisiert: Das Datum, an dem die Übung für die Schüler sichtbar wird. Die Schüler können die Fragen erst ab dem Startdatum sehen.",
+                "batched": "Gruppiert: Das Datum, an dem die Übung für die Schüler sichtbar wird. Die Schüler können die Fragen erst sehen, wenn ein Stapel gestartet wird und sie das Passwort dafür eingeben.",
+                "individual": "Einzeln: Das Datum, an dem die Übung für die Schüler sichtbar wird. Die Schüler können die Fragen zu dem von ihnen gewählten Zeitpunkt sehen."
```

```suggestion
                "individual": "Einzeln: Das Datum, ab dem das Quiz für die Studierenden in der Aufgabenübersicht sichtbar wird. Die Studierenden können ab diesem Zeitpunkt mit der Bearbeitung beginnen."
```
We use the informal `du` in Artemis.
They cannot only see the quiz in the exercise overview, but can also start working on the quiz right away?

---

Thanks! Yes correct, I'll apply the suggested change.